<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_MicMacStepValueAndSpatialResolution</class>
 <widget class="QWidget" name="Widget_MicMacStepValueAndSpatialResolution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_micmacStepValue">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_micmacStepValue">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>52</y>
     <width>136</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>MicMac step value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_spatialResolution">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>85</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_spatialResolution">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>87</y>
     <width>136</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Spatial resolution</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_fileWithSmallTitle">
    <widget class="QLineEdit" name="lineEdit_fileWST">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>8</y>
       <width>421</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_fileSmallTitle">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>11</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fn</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_file">
    <widget class="QLineEdit" name="lineEdit_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>8</y>
       <width>461</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_micmacStepValue_colon">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>52</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spatialResolution_colon">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>87</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_fileWST</tabstop>
  <tabstop>lineEdit_micmacStepValue</tabstop>
  <tabstop>lineEdit_spatialResolution</tabstop>
  <tabstop>lineEdit_file</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
